code/modules/reagents/chemistry/reagents/toxins.dm

```diff
@@ -629,6 +641,7 @@
 	reagent_state = LIQUID
 	color = "#7F10C0"
 	can_synth = 0
+	taste_message = null
```

Comment:
Is this a stealth poison? I mean maybe there won't be enough in a food to taste if it had one but asking still. Maybe it doesn't have a taste either way